README

```diff
@@ -159,18 +166,18 @@ The following table explains the defaults:
 ┃                   │ preëdit text.                                           ┃
 ┠───────────────────┼─────────────────────────────────────────────────────────┨
 ┃ Delete            │ Remove the typed key to the right of the cursor in the  ┃
-┃                   │ preëdit text.                                           ┃
+┃ <kbd>⌫</kbd>      │ preëdit text.                                           ┃
 ┠───────────────────┼─────────────────────────────────────────────────────────┨
 ┃ Control+Delete    │ Remove everything to the right of the cursor in the     ┃
-┃                   │ preëdit text.                                           ┃
+┃ <kbd>⌃⌫</kbd>     │ preëdit text.                                           ┃
```

Comment:
This doesn't  look like Control+Delete. 

By the way, I  autogenerate this section explaining the key bindings  by viewing http://mike-fabian.github.io/ibus-typing-booster/documentation.html#key-bindings with the  w3m text browser to  keep the README and the online documentation in sync.

Reply:
Oh, I see... so what do I do then, should I leave every thing as it is and close the pull request or make some changes?

(*I'm not much familiar with git*)

Reply:
No problem! I can work a bit on this. But not next week, the week after that.
You want to to convert README into README.md and add some markup  to make it look better on github. I think I can use some of your changes, I’ll put all the useful ones into one commit with your name the week after next week.

Reply:
Alright!

Reply:
Alright!